pkgs/by-name/sx/sxwm/package.nix

```
@@ -0,0 +1,37 @@
+{
+	lib,
+	stdenv,
+	fetchFromGitHub,
+	libX11,
+	libXinerama,
+	libXcursor,
+}:
+
+stdenv.mkDerivation rec {
+	pname = "sxwm";
+	version = "1.5";
+
+	src = fetchFromGitHub {
+		owner = "uint23";
+		repo = "sxwm";
+		rev = "v${version}";
+		sha256 = "sha256-CMqVAHrW5oluTmB/DHg65wf8NCSX5fksH/L+GQXZV+o=";
+	};
+
+	buildInputs = [
+		libX11
+		libXinerama
+		libXcursor
+	];
+
+	installPhase = "make install PREFIX=$out";
```

```suggestion
  makeFlags = ["PREFIX=$(out)"];
```
https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-makeFlags